lib/src/utils.dart

```diff
@@ -19,27 +19,26 @@ Stream<String> _byteStreamSplit(Stream<List<int>> stream) =>
 final _timeout = const Duration(minutes: 2);
 final _maxLines = 100000;
 
-ProcessResult runProcSync(String executable, List<String> arguments,
+ProcessResult runProcSync(List<String> arguments,
     {String workingDirectory, Map<String, String> environment}) {
-  log.fine('Running `${[executable, ...arguments].join(' ')}`...');
+  log.fine('Running `${[...arguments].join(' ')}`...');
   return Process.runSync(
-    executable,
-    arguments,
+    arguments.first,
+    arguments.skip(1).toList(),
     workingDirectory: workingDirectory,
     environment: environment,
   );
 }
 
 Future<ProcessResult> runProc(
```

Comment:
This is a breaking change, and should get mentioned in the changelog (IIRC we do use it in pub-dev).

Reply:
Ah good point. I'll do a follow-up preparing for release.